torch/_dynamo/exc.py

```diff
@@ -162,12 +164,11 @@ def __repr__(self) -> str:
 def augment_exc_message(exc, msg="\n", export=False):
     import traceback
 
-    if (
-        hasattr(exc, "real_stack")
-        and len(exc.real_stack) > 0
-        and not (config.verbose and config.suppress_errors)
-    ):
-        msg += f"\nfrom user code:\n {''.join(traceback.format_list(list(reversed(get_real_stack(exc)[0:2]))))}"
```

Comment:
I don't really know why the real stack used to be truncated to two entries; I opt for printing the entire stack because... why would you NOT report the full stack 🤔

Reply:
This case is tested for in test_unsupported_real_stack among others.